airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DbtTransformationWorker.java

```diff
@@ -79,7 +89,7 @@ public void cancel() {
       cancelled.set(true);
       dbtTransformationRunner.close();
     } catch (final Exception e) {
-      e.printStackTrace();
+      LOGGER.error("Unable to cancel Dbt Transformation runner.", e);
```

Comment:
Are we missing a call to `ApmTraceUtils.addExceptionToTrace(e);` here?

Reply:
@colesnodgrass pointed out the same thing.  I have added a `@Trace` to the `cancel` methods and a recording of the exception where applicable.